### PR TITLE
Version 0.2

### PR DESCRIPTION
**Pull request by aimalamiri**
## Update animals table

1. Add species column
2. Add more data to the animal's table
3. Add DELETE and UPDATE queries with TRANSACTION
4. USE GROUP BY and aggregate functions to analyze the table data.

Here are the screenshots of my personal terminal.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -33,4 +34,90 @@ INSERT INTO animals(
...
+  )
+VALUES(
+    7,
+    'Squirtle',
+    '1993-04-02',
+    3,
+    FALSE,
+    12.13,
+    null
+  );
```
Comment (t-yanick) [inline data.sql:83]: - [x] As specified in the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md), the first four values of weight are negative, please kindly fix that.

File: queries.sql [hunk 2]
```diff
@@ -14,3 +14,38 @@ SELECT * FROM animals WHERE neutered=TRUE;
...
+COMMIT;
+SELECT * FROM animals;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT MAX(escape_attempts) FROM animals;
+SELECT name, MAX(escape_attempts) FROM animals GROUP BY name ORDER BY max DESC;
+SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01'GROUP BY species;
```
Comment (t-yanick) [inline queries.sql]: - [ ] As specified in the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md), " `What is the average number of escape attempts per animal type of those born between 1990 and 2000?` ", kindly update your range to include the end of 2000. Some like so: 
```
   WHERE  date_of_birth >= '1990/01/01' AND date_of_birth <= '2000/31/12';
```

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @aimalamiri, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Aimal, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**